manifests/charts/istio-control/istio-discovery/files/injection-template.yaml

```diff
@@ -4,18 +4,26 @@
       requests:
         {{ if (isset .ObjectMeta.Annotations `sidecar.istio.io/proxyCPU`) -}}
         cpu: "{{ index .ObjectMeta.Annotations `sidecar.istio.io/proxyCPU` }}"
+        {{ else }}
```

Comment:
This seems redundant to me since we have:
https://github.com/istio/istio/blob/731c8d02c9043e18850f7c46984ea7d548207121/manifests/charts/istio-control/istio-discovery/files/injection-template.yaml#L30-L32